doc/using/Synthesis.rst

```diff
@@ -58,6 +58,37 @@ See :ghdlsharp:`1174` for on-going discussion about other output formats.
 .. TIP::
    Files can be provided in any order.
 
+.. _synthesis_options:
+
+Synthesis options
+==================
+
+In addition to analyze options, you can use following synthesis specific options:
+
+.. option:: -gNAME=VALUE
+
+  Override top unit generic `NAME` with value `VALUE`.
+
+  Example::
+
+    $ ghdl --synth --std=08 -gDEPTH=12 my_unit
+
+.. option:: --vendor-library=NAME
+
+  Any unit from library NAME is a black box.
+
+  Example::
+
+    $ ghdl --synth --std=08 --vendor-library=vendorlib my_unit
+
+.. option:: --no-formal
+
+  Neither synthesize assert nor PSL.
+
+  Example::
+
+    $ ghdl --synth --std=08 --no-formal my_unit
+
```

Comment:
Would `--no-assert-cover` fit here?

https://github.com/ghdl/ghdl/blob/267ceb3b009f4fa2ccecbfd7ce0f822daa2e2541/src/ghdldrv/ghdlsynth.adb#L129-L130

Reply:
I added documentation of these option as I understand it.

Reply:
Thanks!